search-commons/src/main/java/no/unit/nva/search/ticket/TicketSearchQuery.java

```diff
@@ -141,8 +140,8 @@ protected String[] include() {
   }
 
   @Override
-  public URI openSearchUri() {
-    return fromUri(infrastructureApiUri).addChild(TICKETS, SEARCH).getUri();
+  public URI openSearchUri(String indexName) {
+    return fromUri(infrastructureApiUri).addChild(indexName).addChild(SEARCH).getUri();
```

Comment:
Did you miss the TICKETS part here?

Reply:
indexName provided in input is "tickets" now :)